client/src/App.jsx

```diff
@@ -33,6 +35,20 @@ const AdminResourceManager = lazy(() => import('./pages/AdminResourceManager'));
 
 const App = (): React$Element<React$FragmentType> => {
   const { authed, authRoleIsEquivalentTo } = useAuth();
+  const [language, setLanguage] = useState('English');
+  const [messages, setMessages] = useState({});
+
+  useEffect(() => {
+    const fetchTranslations = async () => {
+      const res = await getTranslationByLanguage(language);
+      let newMessages = {};
+      if (res && res.result) {
+        newMessages = res.result.messages;
+      }
+      setMessages(newMessages);
+    };
+    fetchTranslations();
```

Comment:
I'm *assuming* once we put more translations, it might take longer to switch (we can figure out a loading situation later).  For now, can we trying and short circuit this and just set the messages to be empty if the language is switched to english so we don't need to fetch for that?